### PR TITLE
feat(*): replace and remove dependencies

### LINKED ISSUE #593: [Features] Decrease disk space usage
I've noticed that many glitch users are having trouble with running disc-11. This was caused by the size of node modules that exceeds glitch free plan disk size. We don't want to drop support for glitch users. So, We've come to a conclusion that we need to reduce the disk space used by the project.

To do:
- [x] Replace winston with our own logger
- [x] Remove ts-node
- [x] Remove cli-table
- [x] Replace entities with our own parser
- [x] Replace libsodium-wrappers with tweetnacl
- [x] Install ffmpeg-static in `index.js` instead
- [x] Replace typescript with sucrase

### PR DESCRIPTION
**Pull request by Mednoob**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+const { existsSync, rmSync } = require("fs");
 const { resolve } = require("path");
 const { Server } = require("https");
 
 try {
     require("dotenv/config");
 } catch (err) {
-    console.info("[INFO] It seems dotenv hasn't been installed. Installing...");
-    execSync("npm i dotenv");
+    console.info("[INFO] It seems dotenv hasn't been installed. Re-installing all modules...");
```
Comment (mzrtamp) [inline index.js]:
```suggestion
    console.info("[INFO] It seems dotenv hasn't been installed. Trying to re-install all modules...");
```

File: src/utils/decorators/MusicUtil.ts [hunk 2]
```diff
@@ -1,36 +1,49 @@
 import { createEmbed } from "../createEmbed";
```
Comment (mzrtamp) [inline src/utils/decorators/MusicUtil.ts:2]:
```suggestion
import { CommandContext } from "../../structures/CommandContext";
```

File: src/utils/decorators/MusicUtil.ts [hunk 3]
```diff
@@ -1,36 +1,49 @@
...
-import { Inhibit } from "./Inhibit";
 import i18n from "../../config";
```
Comment (mzrtamp) [inline src/utils/decorators/MusicUtil.ts:3]:
```suggestion
import { createEmbed } from "../createEmbed";
```

File: src/utils/decorators/MusicUtil.ts [hunk 4]
```diff
@@ -1,36 +1,49 @@
...
+import { CommandContext } from "../../structures/CommandContext";
```
Comment (mzrtamp) [inline src/utils/decorators/MusicUtil.ts]:
```suggestion
import i18n from "../../config";
```

File: src/utils/DiscLogger.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { IDiscLoggerOptions } from "../typings";
+import { format } from "date-fns";
+
+enum Colors {
```
Comment (mzrtamp) [inline src/utils/DiscLogger.ts:4]: Sort to this order
> Reset
> Red
> Yellow
> Green
> Blue

**Review by mzrtamp — CHANGES_REQUESTED** [review]

File: src/utils/DiscLogger.ts (hunk 5 above)
Comment (Mednoob) [reply src/utils/DiscLogger.ts:4]: Done in [`e2124bb`](https://github.com/zhycorp/disc-11/pull/594/commits/e2124bbe32e8374cd080c89e6f0ec70062230421)

File: src/utils/decorators/MusicUtil.ts (hunk 4 above)
Comment (Mednoob) [reply src/utils/decorators/MusicUtil.ts]: Done in [`7942706`](https://github.com/zhycorp/disc-11/pull/594/commits/7942706ed4bb19192b9e54a2d87852ffefa4d927)

File: src/utils/decorators/MusicUtil.ts (hunk 3 above)
Comment (Mednoob) [reply src/utils/decorators/MusicUtil.ts:3]: Done in [`7942706`](https://github.com/zhycorp/disc-11/pull/594/commits/7942706ed4bb19192b9e54a2d87852ffefa4d927)

File: src/utils/decorators/MusicUtil.ts (hunk 2 above)
Comment (Mednoob) [reply src/utils/decorators/MusicUtil.ts:2]: Done in [`7942706`](https://github.com/zhycorp/disc-11/pull/594/commits/7942706ed4bb19192b9e54a2d87852ffefa4d927)

**Review by mzrtamp — APPROVED** [review]
Great, thanks!

**Pull request merged**